index.html

```diff
@@ -6,6 +6,7 @@
     <title>Document</title>
 </head>
 <body>
+    <h1>Welcome to Potionfy! Enjoy your stay.</h1>
```

Comment:
could you remove the 'Enjoy your stay' and making the 'Welcome to Potionfy' string on red